youtube_dl/utils.py

```diff
@@ -2371,15 +2371,12 @@ def set_alpn_protocols(ctx):
         return YoutubeDLHTTPSHandler(params, context=context, **kwargs)
 
 
-def bug_reports_message():
+def bug_reports_message(before=';'):
     if ytdl_is_updateable():
         update_cmd = 'type  youtube-dl -U  to update'
     else:
         update_cmd = 'see  https://yt-dl.org/update  on how to update'
-    msg = '; please report this issue on https://yt-dl.org/bug .'
-    msg += ' Make sure you are using the latest version; %s.' % update_cmd
-    msg += ' Be sure to call youtube-dl with the --verbose flag and include its complete output.'
-    return msg
+    return '; please report this issue on https://yt-dl.org/bug .'
```

Comment:
Revert and fix `utils.bug_reports_message()` (2):
```suggestion
    else:
        update_cmd = 'see  https://github.com/ytdl-org/youtube-dl/#user-content-installation  on how to update'

    msg = (
        ' please report this issue on https://github.com/ytdl-org/youtube-dl/issues ,'
        ' using the appropriate issue template.'
        ' Make sure you are using the latest version; %s.'
        ' Be sure to call youtube-dl with the --verbose option and include the complete output.'
    ) % update_cmd

    before = (before or '').rstrip()
    if not before or before.endswith(('.', '!', '?')):
        msg = msg[0].title() + msg[1:]

    return (before + ' ' if before else '') + msg
```